src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -1543,13 +1543,14 @@ public string Description
 
         /// <summary>
         /// The following is the definition of the input parameter "StartupType".
+        /// "StartType" is NativeMethod Get;Set; ServiceController.InputObject.StartType
```

Comment:
Please reword the comment, it is a bit difficult to understand on the first read.

Reply:
@adityapatwardhan 
Will this be ok?

"Set-Service sets ServiceController.InputObject.StartType"

Reply:
@adityapatwardhan . This PR needs your approval.